commands/ps.js

```diff
@@ -13,13 +13,28 @@ function * run (context, heroku) {
 
   let db = yield fetcher.database(app, database)
 
+  let waiting_query = `
+SELECT EXISTS(
+  SELECT 1 FROM information_schema.columns WHERE table_schema = 'pg_catalog'
+    AND table_name = 'pg_stat_activity'
+    AND column_name = 'waiting'
+) AS available
+`
+
+  let waiting_output = yield psql.exec(db, waiting_query)
+  let waiting = 'CASE WHEN wait_event IS NULL THEN false ELSE true END AS waiting'
+
+  if (waiting_output.includes('t')) {
```

Comment:
yeah I don't particularly like this but found similar thing here:
https://github.com/heroku/heroku-pg-extras/blob/07cda801f3368676afd54b6fee8b2874f3487701/lib/util.js#L7-L14

happy to improve if there is simple & nice way. any thoughts?